lib/pdf/writer.rb

```diff
@@ -776,9 +776,9 @@ def find_font(fontname)
 
   def font_file(fontfile)
     path = "#{fontfile}.pfb"
-    return path if File.exists?(path)
```

Comment:
is it just replacing for specific stuff ratefinder related? This is also depreciated 
[https://github.com/metaskills/pdf-writer/blob/7f5bc6c9ce69c26574bbfde36ebdb9ecf06709d0/lib/pdf/techbook.rb#L689        @techbook_code.untaint](https://github.com/konvenit/pdf-writer/blob/b4da24c2ef8a993d425e227a26d0cf7b582bda85/lib/pdf/techbook.rb#L689)

Reply:
yes i only changed the stuff related to ratefinder